backend/main.py

```
@@ -139,4 +139,4 @@ def test_email_config():
 
 
 if __name__ == "__main__":
-    app.run(debug=True)
+    app.run(debug=True, port=5001)
```

Ruff (0.12.2)

Use of `debug=True` in Flask app detected

(S201)